torch/_inductor/codegen/wrapper.py

```diff
@@ -1708,6 +1759,23 @@ def generate_args_decl(self, call_args):
 
         return ", ".join(new_args)
 
+    def generate_default_grid(self, name, grid, cuda=True):
+        if not cuda:
+            return grid
+        assert isinstance(grid, list), f"expected {grid=} to be a list"
+        grid = [e.inner_expr if isinstance(e, SymbolicCallArg) else e for e in grid]
+        grid_fn = default_grid(*grid)
```

Comment:
What is `default_grid`? I cannot find it in the pytorch repo.

Reply:
It's from `from ..triton_heuristics import grid as default_grid`